traffic_portal/test/end_to_end/deliveryServices/pageData.js

```diff
@@ -37,6 +39,10 @@ module.exports = function(){
 	this.protocol=element(by.name('protocol'));
 	this.longDesc=element(by.name('longDesc'));
 	this.remapText=element(by.name('remapText'));
+	this.host=element(by.name('host'));
+	this.typeId=element(by.name('typeId'));
```

Comment:
and maybe rename this one to `staticDNSTypeId` otherwise it looks like it's the delivery service type.

Reply:
Code updated.